sdk/src/main/kotlin/io/rover/sdk/ui/blocks/concerns/text/ViewText.kt

```diff
@@ -24,6 +24,11 @@ internal class ViewText(
         // worry, the ascenders do not appear to be clipped either).
         textView.includeFontPadding = false
 
+        // This is set to prevent the wrong line spacing being used on android 10+ when StyleSpans are applied
+        if (Build.VERSION.SDK_INT >= 29) {
```

Comment:
if there is an upstream bug filed for this, then this workaround may not appropriate for versions after 29.